Makefile.PL

```diff
@@ -21,11 +21,12 @@ requires
   'HTML::Entities'              => 0,
   'JSON::PP'                    => 0,
   'JSON::RPC'                   => 1.01,
-  'JSON::Validator'             => 3.12,
+  'JSON::Validator'             => 4.00,
   'Log::Any'                    => 0,
   'Log::Any::Adapter::Dispatch' => 0,
   'Log::Dispatch'               => 0,
   'LWP::UserAgent'              => 0,
+  'Mojolicious'                 => 7.28,
```

Comment:
maybe add this one to the upgrade script as well ?

Reply:
good point, I've added it.